firebase_functions/functions/src/email.ts

```diff
@@ -1,114 +1,131 @@
-        </div>
-        <div>
-          <p><strong>Newsletter: </strong>   ${packet.newsletter}</p>
-        </div>`,
-    },
-    (error, body) => {
-      if (error) {
-        console.log('Email callback error', error);
-      }
-    }
-  );
+const mailgunSendEmail = async (packet, emailSettings) => {
+  try {
+    const {
+      apiKey, domain, emailRecepient, emailBcc, emailReplyTo, emailSender, emailList,
+    } = emailSettings;
+    const mg = require('mailgun-js')({ apiKey, domain });
 
-  if (packet.newsletter.toString() === 'true') {
-    const list = mg.lists(emailList);
-    const user = {
-      subscribed: true,
-      address: packet.email,
-      name: packet.name,
-      vars: {
-        company: packet.company,
+    mg.messages().send(
+      {
+        from: `Data Marketplace <${emailSender}>`,
+        to: emailRecepient,
+        bcc: emailBcc,
+        'h:Reply-To': packet.email,
+        subject: 'Data Marketplace Form Inquiry',
+        html: `<div>
+            <p><strong>Name: </strong>   ${packet.name}</p>
+          </div>
+          <div>
+            <p><strong>Email: </strong>   ${packet.email}</p>
+          </div>
+          <div>
+            <p><strong>Company: </strong>   ${packet.company}</p>
+          </div>
+          <div>
+            <p><strong>Company Website: </strong>   ${packet.website}</p>
+          </div>
+          <div>
+            <p><strong>Country: </strong>   ${packet.country}</p>
+          </div>
+          <div>
+            <p><strong>Industry: </strong>   ${packet.industry}</p>
+          </div>
+          <div>
+            <p><strong>Category: </strong>   ${packet.category}</p>
+          </div>
+          <div>
+            <p><strong>Message:</strong></p><p>${packet.comments}</p>
+          </div>
+          <div>
+            <p><strong>Newsletter: </strong>   ${packet.newsletter}</p>
+          </div>`,
       },
-    };
-
-    list.members().create(user, (error, data) => {
-      if (error) {
-        console.log('Email members callback error', error);
+      (error) => {
+        if (error) {
+          console.error('Email callback error', error);
+        }
       }
-    });
-  }
+    );
 
-  mg.messages().send(
-    {
-      from: `Data Marketplace <${emailSender}>`,
-      to: packet.email,
-      'h:Reply-To': emailReplyTo,
-      subject: 'Message Received - Data Marketplace',
-      html: `Hi
-        <br/>
-        <br/>
-        Many thanks for your interest in IOTA and inquiry to join the Data Marketplace. The Proof of Concept initiative receives an overwhelming amount of interest and the list of candidates is exceeding our internal capacity to support the integration of all suggested devices.
-        <br/>
-        <br/>
+    if (packet.newsletter.toString() === 'true') {
+      //Add to pending list and send out confirmation email 
+      axios.post('https://newsletter-api.iota.org/api/signup',
+      {
+        email: emailRecepient,
```

Comment:
emailRecepient is a parameter from our configuration. It is set to `...@iota.org`.
What you want is `package.email`, which is user's email address